src/Components/Modal/Modal.scss

```diff
@@ -0,0 +1,139 @@
+$whiteColor: #fff;
```

Comment:
Valdai, kai mano branchas bus sumergintas, main'e atsiras assets folder su _variables.sccs. Tai visus variables reikes perkelt ten, kaip suprantu.

Reply:
Yes, we could do it if we will think that it need to be do.